src/docs/events.yaml

```diff
@@ -0,0 +1,204 @@
+paths:
+  /events/{id}:
+    get:
+      tags: 
+      - "Events"
+      summary: "Get event by id"
+      description: "Returns a single event"
+      operationId: "getEventById"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "ID of the event to return"
+          required: true
+          type: "integer"
+          format: "int64"
+      responses:
+        "200":
+          description: "succesful operation"
+          schema:
+            $ref: "#/components/schemas/Event"
+        "404":
+          description: Invalid event ID supplied, Event not found
+        "500": 
+          description: Internal server error
+components:
+  schemas:
+    Event:
+     type: "object"
+     required:
+       - "title"
+       - "content"
+       - "coverImage"
+       - "date"
+       - "category"
+       - "address"
+       - "location"
+     properties:
+       publisherId:
+        type: "integer"
+        format: "objectId"
+       title:
+        type: "string"
+        description: "Event Title"
```

Comment:
Be consistent with your documentation. Sometimes you have a description, sometimes not. I suggest that for primitive types, instead of a description, you can provide an example.